manifests/moco-nodes.pp

```diff
@@ -1199,3 +1199,13 @@
     $aspects = [ 'low-security' ]
     include toplevel::server
 }
+
+node 't-yosemite-r7-380.test.releng.mdc1.mozilla.com' {
+    $aspects           = [ 'low-security' ]
+    $slave_trustlevel  = 'try'
+    #$pin_puppet_server = 'releng-puppet2.srv.releng.scl3.mozilla.com'
+    #$pin_puppet_env    = 'dcrisan'
```

Comment:
Let's delete these two commented out lines.

Reply:
Removed the commented lines